Makefile

```diff
@@ -10,11 +10,11 @@ DESTDIR ?= /usr/local/bin
 V ?= 1
 
 # Targets
-all: libsnowcrash test-libsnowcrash 
```

Comment:
Because we do not want to build `test-libsnowcrash` while tests are not required.
It still standing as dependency of `test` target. You can use build && run tests via command
`make test`